### ISSUE TITLE
Bad code style in Match for GDScript

### ISSUE THREAD

**Issue by SomethingFawful** [godotengine/godot-docs#1097]
http://docs.godotengine.org/en/3.0/getting_started/scripting/gdscript/gdscript_basics.html#match

```
match x:
    1:      print("We are number one!")
    2:      print("Two are better than one!")
    "test": print("Oh snap! It's a string!")
```
should be
```
match x:
    1:
        print("We are number one!")
    2:
        print("Two are better than one!")
    "test":
        print("Oh snap! It's a string!")
```
as well as the majority of the other match examples

**Comment by SomethingFawful** [comment]
Made a PR for it. #1098

**Comment by mhilbrunner** [comment]
Closed by #1098.
